.teamcity/_self/lib/customBuildType/E2EBuildType.kt

```diff
@@ -56,6 +57,7 @@ open class E2EBuildType(
 	var enableCommitStatusPublisher: Boolean = false,
 	var buildTriggers: Triggers.() -> Unit = {},
 	var buildDependencies: Dependencies.() -> Unit = {},
+	var addWpcomVcsRoot: Boolean = false,
```

Comment:
Just to call it out... I don't love this 😕 I think this E2E abstraction might need some general re-working to accommodate our various different conditions, but that felt out of scope here.

We don't need to add this VCS root to all our builds, so I figured this named boolean param was the way to go for now

Reply:
> I think this E2E abstraction might need some general re-working to accommodate our various different conditions,

Yes, we need to rethink how the abstraction is made. I think it tried to cover too many bases at the expense of being unwieldy to make changes. I agree that it is outside the scope of this particular PR, though I can see us justifying some work on it as part of the Jetpack project.